src/main/java/core/basesyntax/UserService.java

```
@@ -1,8 +1,15 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
-    public int getUserScore(String[] records, String email) {
-        return 0;
+public class UserService {
+    public int getUserScore(String[] records, String email) throws UserNotFoundException {
```

should we throw an exception in the method signature?